readme.md

```diff
@@ -1,115 +1,96 @@
 
-- `read_data_frames(ts_df, lar_df)` 
-	Reads in a HMDA submission test file and outputs a TS and LAR dataframe. 
+For 2018: 
+1. Run `cd 2018/python` to navigate to the python directory. 
+2. Run `python3 generate_2018_error_files.py`
+3. The error files files for testing syntax, validity, and quality edit test files should appear in the following diretories:
+	- Syntax: `2018/edits_files/test_files/{Bank Name}/syntax`
+	- Validity: `2018/edits_files/test_files/{Bank Name}/validity`
+	- Quality: `2018/edits_files/test_files/{Bank Name}/quality`
 
-- `unique_uli(new_lar_df, lei)`
-	Writes a set of unique ULI’s with check digits to a LAR dataframe given an LEI input. 
+Note: the filepath and filename for test files can be adjusted in the [2018 Test Filepaths](https://github.com/cfpb/hmda-test-files/tree/master/2018/configurations/test_filepaths.yaml) YAML configuration file under the keys, `syntax_filepath`, `validity_filepath`, and `quality_filepath`. 
 
-- `new_lar_rows(row_count, lar_df, ts_df)` 
-	Duplicates or retracts rows in a LAR dataframe to produce a file with a certain number of rows. Provides capabilities for creating LAR dataframes with large row counts. 
+## Generating Large Files 
+Often, large files would need to be generated to support load testing. The repository allows users to creates a test file based on existing test file data to create a new file with a larger number of rows.
 
-- `row_by_row_modification(lar_df, yaml_filepath='configurations/row_by_row_modification.yaml')`
-	Changes a LAR dataframe given changes specified in a yaml file. It allows users to test for minor changes in LAR data.
+To generate large files for 2019: 
+1. Run `cd 2019/python` to navigate to the python directory.
+2. Adjust the [2019 File Large File Script Configuration](https://github.com/cfpb/hmda-test-files/tree/master/2019/configurations/test_filepaths.yaml) to specify bank name, lei, tax id, row count, output filepath, and output filename. 
+3. Run `python3 large_test_files_script.py` to produce the large file. 
 
-- `change_bank(ts_data, lar_data, new_bank_name, 
-	new_lei, new_tax_id)`
-	Changes TS and LAR dataframes to a new test institution with a new Institution Name, LEI, and Tax ID. 
+To generate large files for 2018: 
+1. Run `cd 2018/python` to navigate to the python directory.
+2. Adjust the [2018 File Large File Script Configuration](https://github.com/cfpb/hmda-test-files/tree/master/2018/configurations/test_filepaths.yaml) to specify bank name, lei, tax id, row count, output filepath, and output filename. 
+3. Run `python3 large_test_files_script.py` to produce the large file. 
 
-- `check_digit_gen(valid=True, ULI)`
-	 Generates and appends a check digit to a ULI.   
 
 ## Data Generation Notes:
-These values must be changed manually in the config.yaml file
+These values must be changed manually in the config.yaml file in each year directory. 
```

Comment:
Change to:

`The default values for Bank0 are listed below. These files must be changed separately for each year. 

For 2018 use [this configuration file](link) for 2019 use [this configuration file](link).`